src/turnkeyml/build/export.py

```diff
@@ -279,9 +280,31 @@ def fire(self, state: build.State):
         default_warnings = warnings.showwarning
         warnings.showwarning = _warn_to_stdout
 
+        stats = fs.Stats(
+            state.cache_dir, state.config.build_name, state.evaluation_id
+        )
+
+        # Verify if the exported model matches the input torch model
+        try:
+            # The `torch.onnx.verification.find_mismatch()` takes input arguments to the
+            # model as `input_args (Tuple[Any, ...])`
+            export_verification = torch.onnx.verification.find_mismatch(
```

Comment:
How much time does this take? Can you provide some before/after stage durations for common models (rn50, bert)?